src/index.html

```diff
@@ -10,7 +10,30 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body>      
+    <form class="search-bar" data-qa="big"> 
+      <input 
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search-bar__input search-bar__input--big"
```

Comment:
There is no needs to add modifier. Just use basic class search-bar__input.
```suggestion
        class="search-bar__input"
```